test.js

```diff
@@ -284,6 +284,24 @@ test('removes -nocookie', t => {
 	t.is(fn('http://www.youtube-nocookie.com/user/username#p/u/1/ABC12302').service, 'youtube');
 });
 
+/** Unfortunately I was not able to run the tests locally as there were
+64 different syntax errors, but I've run our own tests with the same urls
+**/
+test('removes #t=0m10s ', t => {
+	t.is(fn('https://www.youtube.com/watch?v=G-3YxlZIhus#t=0m10s').id, 'G-3YxlZIhus');
+	t.is(fn('http://www.youtube.com/watch?v=G-3YxlZIhus#t=0m10s').id, 'G-3YxlZIhus');
+	t.is(fn('https://youtube.com/user/WMFinland#p/a/u/1/G-3YxlZIhus').id, 'G-3YxlZIhus');
+
+	t.is(fn('http://www.youtube.com/watch?v=G-3YxlZIhus#t=0m10s').service, 'youtube');
+});
+
+test('removes feature', t => {
+	t.is(fn('http://youtu.be/G-3YxlZIhus&feature=channel').id, 'G-3YxlZIhus');
+	t.is(fn('http://youtube.com/vi/G-3YxlZIhus&feature=channel').id, 'G-3YxlZIhus');
+
+	t.is(fn('http://youtube.com/vi/G-3YxlZIhus&feature=channel').service, 'youtube');
```

Comment:
Let's add a test that looks like this so that we can make sure that whatever routine you use to remove the `feature` parameter doesn't break the search for the videoid

```js
t.is(fn('http://www.youtube.com/watch?feature=foo&v=G-3YxlZIhus#t=0m10s').id, 'G-3YxlZIhus');
t.is(fn('http://www.youtube.com/watch?feature=foo&v=G-3YxlZIhus#t=0m10s').service, 'youtube');
```